build.gradle

```
@@ -19,7 +19,10 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.3'
```

포멧을 일치시킨다면 보기 좋을것같습니다.
```suggestion
    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.3'
```